doc/classes/MultiMesh.xml

```diff
@@ -48,6 +48,14 @@
 				Returns the [Transform2D] of a specific instance.
 			</description>
 		</method>
+		<method name="reset_instance_physics_interpolation">
+			<return type="void" />
+			<argument index="0" name="instance" type="int" />
+			<description>
+				When using [code]physics interpolation[/code], this function allows you to prevent interpolation on an instance in the current physics tick.
```

Comment:
```suggestion
				When using [i]physics interpolation[/i], this function allows you to prevent interpolation on an instance in the current physics tick.
```